threedi_modelchecker/exporters.py

```diff
@@ -41,6 +41,15 @@ def format_check_results(check: BaseCheck, invalid_row: NamedTuple):
 
 
 # check overview export functions
+def order_checks(checks) -> list:
+    """
+    Alphabetically order checks so that they will consistently be ordered the same.
```

Comment:
Please mention that sorting is done by error code, independent of error level (warning or error).

Reply:
done